src/applications/appeals/shared/content/contestableIssues.jsx

```diff
@@ -113,7 +113,7 @@ export const noneSelected =
  */
 export const NoneSelectedAlert = ({ count, headerLevel = 3, inReviewMode }) => {
   const wrapAlert = useRef(null);
-  const Header = `H${headerLevel}`;
+  const Header = `h${headerLevel}`;
```

Comment:
Good catch on the 'h'. Recently saw that the capital H doesn't render the header correctly.